### PR TITLE
Offset fullscreen panel for app bar

### PR DESCRIPTION
**Pull request by foxymiles**
**User-Facing Changes**
None (feature flagged)

**Description**
Offset position of fullscreen panel by app bar height so it doesn't cover the app bar.

### CODE REVIEW

File: packages/studio-base/src/components/PanelRoot.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const usePanelRootStyles = makeStyles<
     entered: {
       borderWidth: 4,
       position: "fixed",
-      top: 0,
+      top: sourceRect?.top ?? 0,
```
Comment (jtbandes) [inline packages/studio-base/src/components/PanelRoot.tsx:70]: I don't think this is right, sourceRect should be where the panel is normally located on screen.

Comment (jtbandes) [reply packages/studio-base/src/components/PanelRoot.tsx:70]: https://user-images.githubusercontent.com/14237/236363369-3b50348e-9350-4df2-bfaa-bd245c614805.mov

Comment (foxymiles) [reply packages/studio-base/src/components/PanelRoot.tsx:70]: Yeah you're right that's only correct for a panel at the top of the layout.

Comment (foxymiles) [reply packages/studio-base/src/components/PanelRoot.tsx]: Changed this to use the app config value explicitly. Unfortunately this meant hardcoding the height of the app bar but we're already also doing that for the playback controls.

Comment (jtbandes) [reply packages/studio-base/src/components/PanelRoot.tsx]: I think we have a constant for it somewhere. We use it for the electron desktop window too.

Comment (foxymiles) [reply packages/studio-base/src/components/PanelRoot.tsx]: Yep found it.

**Review by jtbandes — APPROVED** [review]

**Pull request merged**